news-app-diptimehta/src/components/headlines/HeadlineCard.jsx

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import "./HeadlineCard.css";
+export const HeadlineCard = (props) => {
```

Comment:
destructure this property as
```suggestion
export const HeadlineCard = ({ headline }) => {
```